### PR TITLE
Feature | Read Only Permissions

### PR DESCRIPTION
**Pull request by ChazUK**

### CODE REVIEW

**Comment by ChazUK** [comment]
[(13) Site-wide - Read Only User permissions](https://trello.com/c/4NczPkjf/136-13-site-wide-read-only-user-permissions)

File: src/app/features/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -23,6 +25,8 @@ export class DashboardComponent implements OnInit {
   ) {}
 
   ngOnInit() {
+    this.canViewStaffRecords = this.userService.loggedInUser.role !== Roles.Read;
```
Comment (omarCreativeDev) [inline src/app/features/dashboard/dashboard.component.ts]: would it be safer to do this `this.canViewStaffRecords = this.userService.loggedInUser.role === (Roles.Admin || Roles.Edit);` ?

File: src/app/features/workplace/view-workplace/view-workplace.component.ts [hunk 2]
```diff
@@ -52,6 +58,10 @@ export class ViewWorkplaceComponent implements OnInit, OnDestroy {
 
   public onDeleteWorkplace(event: Event): void {
     event.preventDefault();
+    if (!this.canDelete) {
```
Comment (omarCreativeDev) [inline src/app/features/workplace/view-workplace/view-workplace.component.ts]: is this canDelete check needed here? i would have thought the CTA would be hidden with an ngIf with this canDelete?

File: src/app/features/dashboard/dashboard.component.ts (hunk 1 above)
Comment (ChazUK) [reply src/app/features/dashboard/dashboard.component.ts]: done

File: src/app/features/workplace/view-workplace/view-workplace.component.ts (hunk 2 above)
Comment (ChazUK) [reply src/app/features/workplace/view-workplace/view-workplace.component.ts]: It's just a safety precaution.

Comment (omarCreativeDev) [reply src/app/features/workplace/view-workplace/view-workplace.component.ts]: cool

Comment (ChazUK) [reply src/app/features/workplace/view-workplace/view-workplace.component.ts]: removed

**Pull request merged**